### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Sajeelzafar**
### In this pull request, the following tasks have been performed:

- Styled navbar according to the given design.
- Styled list items according to the given design.
- Styled add book according to the given design.
- Add book fields refresh whenever a new book is added, recommended by the previous reviewer.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,31 +1,40 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
 
 const Navbar = () => {
   const links = [
     {
       id: 1,
       path: '/books',
```
Comment (danielufeli) [inline src/components/Navbar.js:10]: - [x] Nice working on your navigation, Kindly check when the root `/` page loads it shows only the header, you can make the `/books` route the root page `/` to solve this.

File: src/components/Books.js [hunk 2]
```diff
@@ -13,16 +13,28 @@ const Books = () => {
   }, []);
   return (
     <div>
-      List of Books will be displayed here:
-
       {
-            book.map((element) => (
+      book.map((element) => (
```
Comment (danielufeli) [inline src/components/Books.js:18]: - [x] Nice working on your design, kindly adjust the book section to look exactly like what was given in the template. see the screenshot below:

### Your Design
your-img

### Template

tep-img

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Sajeelzafar , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @Sajeelzafar, 
congratulations

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions
- [x] For the next project kindly use the new Readme template
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**